src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php

```diff
@@ -69,20 +69,34 @@ protected function setup_repository_args( Context $context = null ) {
 		$context = null !== $context ? $context : $this->context;
 
 		$args = parent::setup_repository_args( $context );
+		$args['ends_after'] = 'now';
 
 		return $args;
 	}
 
 	/**
-	 * Overrides the base View method to fix the order of the events in the `past` display mode.
+	 * Overrides the base View method.
 	 *
-	 * @since 5.2.1
+	 * @since TBD
 	 *
-	 * @return array<string,mixed> The List View template vars, modified if required.
+	 * @return array<string,mixed> The Widget List View template vars, modified if required.
 	 */
 	protected function setup_template_vars() {
 		$template_vars = parent::setup_template_vars();
 
-		return $template_vars;
+		// Here update, add and remove from the default template vars.
+		$template_vars['view_more_link']   = tribe_get_events_link();
+		$template_vars['widget_title']     = $this->context->get( 'widget_title' );
+		$template_vars['show_latest_past'] = false;
+
+		/**
+		 * Applies a filter to the template vars for the List Widget.
+		 *
+		 * @since TBD
+		 *
+		 * @param array<string,mixed> $template_vars The updated instance of the widget.
+		 * @param \Tribe__Context $context   Tribe context used to setup the view.
+		 */
+		return apply_filters( 'tribe_events_view_v2_list_widget_template_vars', $template_vars, $this->context );
```

Comment:
Isn't this value already filtered in the Abstract class in Common in the `filter_template_args` method?
I would remove this one as it seems to be redundant.

Reply:
There is this filter in the View class: 
tribe_events_views_v2_view_{$this->slug}_template_vars

I removed the filter here and instead will use that in Pro to modify the template vars.